### DISCUSSION TITLE
[Support]: Events not in the correct order

### DISCUSSION THREAD

**Discussion by rayzorben** [blakeblackshear/frigate#10243] [General Support]
### Describe the problem you are having

Take this event timeline. As you can see, the car driving away event happened before the person on the porch event. Even though it was this person leaving in that car.

![image](https://github.com/blakeblackshear/frigate/assets/7737834/2c4b531c-e8c2-4c7f-9f8e-836ec7ebe080)

However if you look at the timestamps from the same porch camera, the person on porch in reality happened before he drove away in his car, but the timeline reflects otherwise

![image](https://github.com/blakeblackshear/frigate/assets/7737834/544e0b58-6553-41b5-9b67-9dbaf20b4e27)
![image](https://github.com/blakeblackshear/frigate/assets/7737834/dbbbf722-eeb8-46ee-9aff-5d69e960bb73)

### Version

0.13.2-6476f8a

### Frigate config file

```yaml
NR
```

### Relevant log output

```shell
BR
```

### FFprobe output from your camera

```shell
BR
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Amcrest

**Comment by NickM-27** [comment]
this is working as expected. The snapshots are taken from different points in time, the car was detected and being tracked before the person walked out

**Reply by rayzorben** [reply]
Honestly I don't like that and it doesn't make sense. The timeline should reflect the time of the event not the time that it started tracking. I get what you are saying but it doesn't even make sense as the event of being seen on the street isn't part of the tracking on the driveway. There are 2 unique events from the perspective of the street - a car coming, and a car going.

![image](https://github.com/blakeblackshear/frigate/assets/7737834/21285f03-2e55-4bce-97a7-f9c42559c276)

**Reply by NickM-27** [reply]
this is the way that it works in 0.13. I understand what you are saying, frigate 0.14 / the new UI will have combined the person and car and displayed it as a single event (person + car) in the new UI
